plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/BridgingHiveMetastore.java

```diff
@@ -278,7 +278,14 @@ public void createTable(Table table, PrincipalPrivileges principalPrivileges)
     @Override
     public void dropTable(String databaseName, String tableName, boolean deleteData)
     {
-        delegate.dropTable(databaseName, tableName, deleteData);
+        try {
```

Comment:
`Ignore internal TableNotFoundException in DROP TABLE for ThriftHive` -> `Ignore internal TableNotFoundException in DROP TABLE`